parent/pom.xml

```diff
@@ -4040,6 +4040,68 @@
         </dependencies>
     </dependencyManagement>
 
+    <dependencies>
```

Comment:
We should in another PR see what of this crap we can get rid of

Reply:
+1. I created CAMEL-17435 so I don't forget to review those (for now, I mostly did a 1-to-1 translation of the jdk9+ profiles that were activated by default). I'll take care of that tomorrow, after I come back from a short PTO.